src/app/meds-n-labels/activity-1/activity-1.component.ts

```diff
@@ -198,7 +211,7 @@ export class MatchmedsComponent implements OnInit {
      * @param  {number} $event Mouse Coordinates
      */
     mouseClick($event) {
-        this.setLineStyle('#9932cc', 10);
+        this.setLineStyle('#DC5034', 10);
```

Comment:
let's save this as a String const and use it.

Reply:
Done.